frontend/app/models/user.js

```diff
@@ -42,7 +42,11 @@ export default Model.extend({
       .map(g => g.split('-')[0])
   }),
 
+  admin: computed('groups.[]', function() {
+    return this._checkGroup(window.CustomerCenter.adminGroup)
```

Comment:
Please don't use a global variable. Set some kind of config property for this

Reply:
:+1: